src/modules/aws.rs

```diff
@@ -47,7 +47,7 @@ fn get_aws_region_from_config(context: &Context, aws_profile: Option<&str>) -> O
 }
 
 fn get_aws_profile_and_region(context: &Context) -> (Option<Profile>, Option<Region>) {
-    let profile_env_vars = vec!["AWSU_PROFILE", "AWS_VAULT", "AWS_PROFILE"];
+    let profile_env_vars = vec!["AWSU_PROFILE", "AWS_VAULT", "AWSUME_PROFILE", "AWS_PROFILE"];
```

Comment:
Why this specific order? Are they intentionally ordered this way?

Reply:
I used the existing order to determine this.

In my case if there is an AWSUME_PROFILE defined that means that AWSUME has ran and set the credentials to use. I would assume that will be the same for AWS_VAULT and AWSU_PROFILE which do similar things to AWSUME.

Reply:
I get that my question was more like why `AWSU_PROFILE` comes before `AWSUME_PROFILE` and why `AWS_VAULT` has higher search priority than the latter. I guess we are safe to assume that users won't have multiple AWS credential managers installed so we are good even the search order seems pretty random.

Reply:
Ahh. I would say the ordering is probably based on the order added. Personally it should be any credential manager then the AWS one. Most people will only use a single credential manager so it won't matter but if you have more than one then I don't think the ordering will help either as you would need to know which one was executed last.